src/Workspaces/VisualBasic/Portable/Extensions/SyntaxTreeExtensions.vb

```diff
@@ -85,6 +85,17 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions
         End Function
 
         <Extension()>
+        Public Function IsEntirelyWithinStringLiteral(syntaxTree As SyntaxTree, position As Integer, cancellationToken As CancellationToken) As Boolean
```

Comment:
What does/should this return if `position` is an in an interpolated string expression hole?